GeneralsMD/Code/GameEngine/Source/GameLogic/Object/Body/UndeadBody.cpp

```diff
@@ -81,7 +81,11 @@ void UndeadBody::attemptDamage( DamageInfo *damageInfo )
 
 	if( damageInfo->in.m_damageType != DAMAGE_UNRESISTABLE
 			&& !m_isSecondLife
+#if RETAIL_COMPATIBLE_CRC
```

Comment:
I think better use RETAIL_COMPATIBLE_BUG here

Reply:
As in this is a desirable bug to be kept? If some players strangely desire this behaviour, they can surely just tweak the Battle Bus's `HAZARD_CLEANUP` damage modifier to deal damage.
```diff
Armor BattleBusTruckArmor
+  Armor = HAZARD_CLEANUP      50%    ;Now consistently harmed by cleaning weapons
-  Armor = HAZARD_CLEANUP       0%    ;Not harmed by cleaning weapons
End
```

Reply:
Ok